draft-ietf-rats-corim.md

```diff
@@ -1107,6 +1110,16 @@ checks, and trust anchor checks.
 {::include cddl/crypto-key-type-choice.cddl}
 ~~~
 
+##### Integrity Registers {#sec-comid-integrity-registers}
+
+An Integrity register is a group of one or more named measurements.
```

Comment:
```suggestion
Integrity registers is a group of one or more named measurements.
```

Reply:
reworded in [0daa574](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/189/commits/0daa57453d485e1ad42dd2d338868d0b3d389fcd)